apps/meteor/client/views/admin/workspace/components/WorkspaceCardSectionTitle.tsx

```diff
@@ -0,0 +1,16 @@
+import { Box } from '@rocket.chat/fuselage';
+import React from 'react';
+
+type WorkspaceCardSectionTitleProps = {
+	variant?: 'p2b' | 'h4';
+	title: string;
+	titleToContentGap?: number;
+};
+
+const WorkspaceCardSectionTitle = ({ variant = 'p2b', title, titleToContentGap = 0 }: WorkspaceCardSectionTitleProps) => (
```

Comment:
If the variations using `h4` all have the `titleToContentGap={20}` they can be a different component, or you can simply add the margin if the variant is `h4`. 
Again, we want this components to be as much standardized as much as we can - so we are avoiding opening those styling props.